components/hero/hero.snip.html

```diff
@@ -0,0 +1,20 @@
+{{#hero}}
```

Comment:
Discussed this with few others , i dont think we will be using a component for this , lets move it to the template for now and call it 

`<template-namepsace>-hero-item `